drivers/sensor/nrf5/temp_nrf5.c

```diff
@@ -112,7 +112,7 @@ static int temp_nrf5_init(const struct device *dev)
 {
 	struct temp_nrf5_data *data = dev->data;
 
-	LOG_DBG("");
```

Comment:
please just delete that, not sure the reason it's actually here.

Reply:
Done.